src/blocks/post-carousel/inspector.js

```diff
@@ -69,10 +88,8 @@ class Inspector extends Component {
 					<RangeControl
 						label={ __( 'Columns', 'coblocks' ) }
 						value={ columns }
-						onChange={ ( value ) => {
-							setAttributes( { columns: value } );
-						} }
-						min={ 2 }
+						onChange={ ( value ) => columnCountByPosts( value ) }
+						min={ Math.min( 2, postCount ) }
```

Comment:
I don't think a minimum of `2` columns is necessary here. This might have been missed previously so let's fix it here. We have defaults for setting 1 or 2 columns based on the style if the value was not explicitly changed. One column is perfectly acceptable.

Reply:
> I don't think a minimum of 2 columns is necessary here. This might have been missed previously so let's fix it here. We have defaults for setting 1 or 2 columns based on the style if the value was not explicitly changed. One column is perfectly acceptable.

I would appreciate a bit of clarification here. `Math.min( 2, postCount )` should be either 1 or 2 at lowest value depending on the style of posts-carousel (number of pooled posts). Is there a preferred way to accomplish this?